ansible/roles/dokuwiki/files/initialize.d/dokuwiki

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import random
+from pathlib import Path
+from datetime import datetime
+
+FILES = {}
+
+FILES['acl.auth.php'] = """\
```

Comment:
I really want us to use separate files and jinja on everything, because this is getting ugly really fast. Would it be make sense to move them under systeml once it's merged?